### DISCUSSION TITLE
YEARS BADGE

### DISCUSSION THREAD

**Discussion by imDMK** [community/community#67812] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I suggest adding badges for account length.
For example, someone who has an account for 5 years receives a 5-year badge.

**Comment by vishalkrishna8** [comment]
Badges for account length is a great idea, don't think there is any such feature on GitHub right now.

**Comment by coreybutler** [comment]
This is a great idea. Heck, I'd like to see a stat on each repo showing the repo's age as well.

**Comment by flat-life** [comment]
great idea i want to see it as soon as possible

**Comment by mkdirer** [comment]
me too
